modules/dataframe/include/inviwo/dataframe/datastructures/column.h

```diff
@@ -321,10 +321,14 @@ namespace detail {
 template <typename T, typename std::enable_if<std::is_integral<T>::value, int>::type = 0>
 void add(Buffer<T>* buffer, const std::string& value) {
     T result;
-    std::istringstream stream(value);
-    stream >> result;
-    if (stream.fail()) {
-        throw InvalidConversion("cannot convert \"" + value + "\" to target type");
+    if (value.empty()) {
+        result = T{0};  // no special value indicating missing data for integral types
```

Comment:
Why set 0 if the data is missing?

Reply:
What else should we use for int? We are tied to integer values due to the use of `Buffer`. Also, the CSV reader now distinguishes between float and int columns. One reason for this is that joining DataFrames columns with different types will be interesting to implement. For now, only matching columns (same title, same format) are handled.

Reply:
std::numeric_limits<T>::quiet_NaN
(Which will result in 0 anyway though)

Reply:
hmm. Needs discussion. For now, there is a special overload for floating point values when setting data values from `std::string`.